app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,116 @@
+:root {
+  --blue: #3778c2;
+  --green: #5fb523;
+  --ash: #434b54;
+  --black: #555e77;
+  --white: #fff;
+}
```

Comment:
- [ ] As you did a great job with the design, please modify it to match the given design as per [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow%20these%20design%20guidelines%2C%20including%3A). 🙏